### ISSUE TITLE
[BUG] a11y: header section - Menu label is verbose - Short description of the bug

### ISSUE THREAD

**Issue by GoCHacpie** [wet-boew/wet-boew#9410]
## Required information

**Describe the bug**
[Issue]
When focus moves on the Menu button, screen reader announces the instructions to access the menu items. There is no need to provide the access instructions as the provided keyboard support is standard and known to assistive technology users. 

[User Impact]
Overly verbose labels are often annoying to screen reader users.

[Code Reference]
<button _ngcontent-neo-c79="" type="button" aria-haspopup="true" aria-expanded="false" aria-label="Press the SPACEBAR to expand or the escape key to collapse this menu. Use the Up and Down arrow keys to choose a submenu item. Press the Enter or Right arrow key to expand it, or the Left arrow or Escape key to collapse it. Use the Up and Down arrow keys to choose an item on that level and the Enter key to access it." tabindex="0"><span _ngcontent-neo-c79="" class="wb-inv">Main </span>Menu <span _ngcontent-neo-c79="" class="expicon glyphicon glyphicon-chevron-down"></span></button>

[Recommendation]
Remove aria-label used on the <button> element. An inner text for button, "Main menu" is sufficient to access the menu. 

Not an accessibility failure. More of a accessibility-usability experience issue.
Severity: 2/10

**To Reproduce**
Steps to reproduce the behaviour:
1. Turn on screen reader NVDA with Chrome/Windows and navigate with keyboard Tab key to Menu button'
2. Listen to super lengthy announcement.  Screen reader user must listen to this lengthy announcement everytime that navigate in the menu.

**Expected behaviour**
recommended use of short label = "Main menu"

**Who shall do the work?**
not sure.

## Additional information (optional)

**Version of WET-BOEW/GCWeb you are using**
e.g. v4.0.36

**Desktop/Smartphone (please complete the following information)**
Browser chrome/firefox/safari/edge

**Screenshots**
If applicable, add screenshots to help explain the bug.
![image](https://user-images.githubusercontent.com/74194190/184948511-4ab0aa66-39c1-4f59-a8c0-3738c1d0c05b.png)

**Additional context**
2 corporate accessibility consultants recommend keeping expandable menu button label simple. However, if this long explanation text was added due to a usability study and an explicit request to add, then keep. This would be surprising though.
